src/environments/environment.ts

```diff
@@ -15,6 +15,7 @@ export const environment = {
     redirectSignIn: 'http://localhost:4200/',
     redirectSignOut: 'http://localhost:4200/',
   },
+  signalLink: 'https://signal.group/#CjQKIM3ulwlm8OT0_dlVj3GRuZNypumHPahTkJxBCQHtoJSmEhDBzZwFxv4lARp7rG-2vF_p'
```

Comment:
You would have to add it in all of the environment.xyz.ts files, as they are replacing this file during the builds for the different targets